web/vtadmin/src/components/inputs/Select.tsx

```diff
@@ -135,6 +137,7 @@ export const Select = <T,>({
     return (
         <div className={containerClass}>
             <Label {...getLabelProps()} label={label} />
+            {!!description && <div className="mt-[-4px] mb-4">{description}</div>}
```

Comment:
I don't think you need the `!!` here.

```suggestion
            {description && <div className="mt-[-4px] mb-4">{description}</div>}
```